src/vue/fields/FileField.vue

```diff
@@ -56,32 +56,31 @@ export default {
   },
   methods: {
     onChange (event) {
-      const file = FileHelper.deriveFileFromChangeEvent(event)
-
-      if (file) {
-        if (!this.isValidFileSize(file)) return
-
-        this.fileUrl = file.name
-        this.$emit('input', new DocumentContainer({
-          mimeType: file.type,
-          type: this.type,
-          name: file.name,
-          file: file
-        }))
-      } else {
-        this.clear()
+      let file
+      try {
+        file = FileUtil.getFileFromEvent(event)
+      } catch (e) {
+        Bus.error('file-field.file-not-selected')
```

Comment:
bad error handling: it’s not so obvious that `getFileFromEvent` throws an error only when the file event is empty